src/rt-app.c

```diff
@@ -1141,6 +1232,9 @@ void *thread_body(void *arg)
 		struct timespec t_diff, t_rel_start;
 
 		set_thread_affinity(data, &pdata->cpu_data);
+#if HAVE_LIBNUMA
```

Comment:
I suspect it might be cleaner to put the #if HAVE_LIBNUMA into the body of the function set_thread_membind() instead.  This way we would simply return if there is nothing to set or if there is something to set and HAVE_LIBNUMA is not defined, we can simply exit failure.